src/cockpit/389-console/src/lib/security/certificateManagement.jsx

```diff
@@ -6,10 +6,12 @@ import {
     TabContainer,
     TabContent,
     TabPane,
-    Button,
     Spinner,
     noop
 } from "patternfly-react";
+import {
+    Button
```

Comment:
Couple of 'style lines' are missed in this file:

```
lib/security/certificateManagement.jsx-534    bsStyle="primary"
lib/security/certificateManagement.jsx-553    bsStyle="primary"
```

Reply:
Hi @droideck 
I closed this PR because I made a mess of a merge conflict using the git web tool. I rebased and created a new PR for this change: https://github.com/389ds/389-ds-base/pull/4745